manifests/server/mysqltuner.pp

```diff
@@ -8,7 +8,7 @@
 
   if $source {
     $_version = $source
```

Comment:
Does that change not need to go on the `$_version` line rather than the `$_source` line? It appears that the version is the part that breaks.